content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -169,7 +169,8 @@ different Kubernetes components.
 | `ServiceNodeExclusion` | `true` | Beta | 1.19 | |
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
 | `SetHostnameAsFQDN` | `false` | Alpha | 1.19 | 1.19 |
-| `SetHostnameAsFQDN` | `true` | Beta | 1.20 | |
+| `SetHostnameAsFQDN` | `true` | Beta | 1.20 | 1.21 |
+| `SetHostnameAsFQDN` | `true` | GA | 1.22 | - |
```

Comment:
This whole block must move to the section for graduated feature gates.

Reply:
thanks @sftim , I will update